src/components/flash/flash.js

```diff
@@ -305,19 +318,19 @@ class Flash extends React.Component {
           key={ title }
           title={ title }
           open={ this.state.dialogs[title] || false }
-          onCancel={ this.toggleDialog.bind(this, title) }
+          onCancel={ this.toggleDialog(title) }
         >
           { desc }
         </Alert>
       );
 
       // create text for item
-      text = (
+      return (
```

Comment:
@toni-leigh I've tweaked this function so now it only has a single return statement at the bottom. Note I need to set the param to a new variable using `let` to allow it to be redefined inside the if statement.

So taking the two previous examples I wrote above, this is actually a third one I've implemented:
**Option 3**
```
function(y) {
  let x = y
  if (y === 137) {
    x = 'yep'
  }
  return x;
}
```

I must admit I find option #2 the clearest, but maybe that's me. This probably seems like a minor refactoring, but I suspect that params are redefined quite a lot throughout the code so I may need to do this a fair bit.